fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb

```diff
@@ -34,7 +34,24 @@ def run!
       parse_global_options
       remove_global_options options, @args
 
+      xcode_outdated = false
       begin
+        unless FastlaneCore::Helper.xcode_at_least?(Fastlane::MINIMUM_XCODE_VERSION)
+          xcode_outdated = true
+        end
+      rescue
+        # We don't care about exceptions here
+        # We'll land here if the user doesn't have Xcode at all for example
+        # which is fine for someone who uses fastlane just for Android project
+        # What we *do* care about is when someone links an old version of Xcode
+      end
+
+      begin
+        if xcode_outdated
```

Comment:
> +          # We have to raise that error within this `begin` block to show a nice user error without a stack trace

Mh, yes, I think that's good though? This way we'll show a nice error message without stack trace.